### DISCUSSION TITLE
Reflex Office Hours: Thursdays 4pm PT

### DISCUSSION THREAD

**Discussion by Alek99** [reflex-dev/reflex#2836] [Announcements]
Come hang out with the Reflex team!

This is an open session in our [Discord](https://discord.gg/T5WSbC2YtQ) where you can ask for help on Reflex, learn how to get started contributing, or ask about the Reflex Roadmap.

https://discord.com/events/1029853095527727165/1216897937909485568

**Comment by TimChild** [comment]
Does this still happen? I tried clicking on the link, and the `Office Hours` thing in discord, but not much seemed to happen.
